server/athenian/api/models/web/release_metric_id.py

```diff
@@ -0,0 +1,7 @@
+from athenian.api.models.web.base_model_ import Enum, Model
+
+
+class ReleaseMetricID(Model, metaclass=Enum):
+    """Linear release metric identifier."""
+
+    RELEASE_COUNT = "release-count"
```

Comment:
Is it possible to differentiate the count for the 2 release strategies: one count for the GH tags and another one for the releases into branch?

Reply:
For the same repository?

Reply:
Or do you mean that the spec's response should include the applied release strategy?

Reply:
The release strategy yes. I'm thinking that some users will want to know the number of release GH tags, without counting the other releases from the release into a branch strategy.

Reply:
ok :+1: